File: DayTrades/Pick.m
//
//  Pick.m
//  DayTrades
//
//  Created by Jason Wells on 1/9/15.
//  Copyright (c) 2015 Jason Wells. All rights reserved.
//

#import "Pick.h"

@implementation Pick

@dynamic user;
@dynamic account;
@dynamic dayOfTrade;
@dynamic symbol;
@dynamic open;
@dynamic close;
@dynamic value;
@dynamic change;
@dynamic processed;

+ (NSString *)parseClassName {
    return @"Pick";
}

- (id)initForAccount:(Account *)account withSymbol:(NSString *)symbol withDayOfTrade:(NSString *)dayOfTrade {
    if (self = [super init]) {
        self.account = account;
        self.user = account.user;
        self.symbol = symbol;
        self.dayOfTrade = dayOfTrade;
    }
    return self;
}

@end

File: DayTrades/Pick.swift
//
//  Pick.swift
//  DayTrades
//
//  Created by Jason Wells on 7/12/15.
//  Copyright (c) 2015 Jason Wells. All rights reserved.
//

import Foundation

class Pick: PFObject, PFSubclassing {
    
    @NSManaged var user: PFUser;
    @NSManaged var account: Account;
    @NSManaged var symbol: String;
    @NSManaged var dayOfTrade: String;
    @NSManaged var open: Double;
    @NSManaged var close: Double;
    @NSManaged var value: Double;
    @NSManaged var change: Double;
    @NSManaged var processed: Bool;
    
    convenience init(account: Account, symbol: String, dayOfTrade: String) {
        self.init()
        self.user = account.user
        self.account = account
        self.symbol = symbol
        self.dayOfTrade = dayOfTrade
    }
    
    static func parseClassName() -> String {
        return "Pick"
    }
    
}